dynamic_programming/word_break.py

```diff
@@ -0,0 +1,107 @@
+"""
+Author  : Alexander Pantyukhin
+Date    : December 12, 2022
+
+Task:
+Given a string s and a dictionary of strings wordDict,
+return true if s can be segmented into a space-separated
+sequence of one or more dictionary words.
```

Comment:
It is a list of `words` so it is confusing for the reader if we call it a dict.
```suggestion
Given a string and a list of words, return true if the string can be
segmented into a space-separated sequence of one or more words.
```